### PR TITLE
Ra dspdc 427 init containers

### PR DESCRIPTION
**Pull request by raaidbroad**
move relevant code for deployment init containers next to app code, comingle docker image tagging

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -5,12 +5,17 @@ pipeline {
     options {
         timestamps()
         ansiColor('xterm')
-        disableConcurrentBuilds()
     }
     environment {
         PATH = "${tool('sbt')}:$PATH"
     }
     stages {
+        stage('Sanity-check') {
```
Comment (danxmoran) [inline Jenkinsfile]: Oops, this stage should go away now that it's working

File: project/TransporterDeployPlugin.scala [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import com.typesafe.sbt.SbtNativePackager.autoImport.NativePackagerHelper
+import com.typesafe.sbt.packager.NativePackagerKeys
+import com.typesafe.sbt.packager.docker.{Cmd, DockerChmodType, DockerPlugin}
+import com.typesafe.sbt.packager.linux.LinuxKeys
+import com.typesafe.sbt.packager.universal.UniversalPlugin.autoImport._
+import sbt._
+import sbt.Keys._
+
+object TransporterDeployPlugin extends AutoPlugin with LinuxKeys with NativePackagerKeys {
```
Comment (danxmoran) [inline project/TransporterDeployPlugin.scala]: Could you add some scaladoc to this object explaining what it does?

File: Jenkinsfile [hunk 3]
```diff
@@ -28,10 +27,48 @@ pipeline {
...
+                            'aws-to-gcp-agent',
+                            'aws-to-gcp-agent-deploy',
+                            'echo-agent',
+                            'manager',
+                            'manager-deploy',
+                            'manager-migrations'
+                    ]
+
+                    def steps = [
+                            '#!/bin/bash',
```
Comment (benjamincarlin) [inline Jenkinsfile]: Why no pipefail?

Comment (benjamincarlin) [reply Jenkinsfile]: @danxmoran

Comment (danxmoran) [reply Jenkinsfile]: Oversight, will add it in

**Pull request merged**